.github/workflows/windows-build-smoke-test.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         run: | 
           git clone --recursive https://github.com/microsoft/ntosebpfext.git
           cd ${{ env.TEMP }}\ntosebpfext
-          git checkout e7dc209a8be0da2ff5d75f5772a0ee0bf4a10383
+          git checkout tags/v0.6.0
```

Comment:
Can we retain the usage of the commit hash instead?
Should be b4b240c99321f43b073d792a2f8c194e46a1b6ce

Reply:
I ask for my education: What is the advantage of using commit hashes instead of tags ?